ex/auto/writer/odev_build_form/build_form.py

```diff
@@ -84,7 +84,12 @@ def __init__(self, db_path: Path) -> None:
         Lo.close_office()
 
     def create_form(self, doc: XTextDocument) -> None:
-        props = Forms.add_labelled_control(doc=doc, label="FIRSTNAME", comp_kind=Forms.CompenentKind.TextField, y=11)
+
+# Form has four sections: text, command_button, list_box, grid_control
+# Section 1 has two columns
+        DOC = BuildForm.doc
```

Comment:
Change this to _doc or something like that. Local scoped variables should not be upper-case in python. Don't use `doc` in this case because the class level var is already `doc`.

The reason doc was put at the class level is mainly so listeners would not have to depend on a class instance var. Also the original java was this way.
```suggestion
        _doc = BuildForm.doc
```